### DISCUSSION TITLE
Avoid screen lock in presentation mode

### DISCUSSION THREAD

**Discussion by datenschvtz** [sumatrapdfreader/sumatrapdf#4647] [General]
Is it possible to avoid screen lock (or screen saver) in presentation mode? If I show my presentation the screen saver block after x minutes the screen. That ist not very comfortable.

**Comment by GitHubRulesOK** [comment]
Unlikely as it is an application external to SumatraPDF so would lock any other similar application, I am "guessing" that without you debug why, the fact that keyboard is focused inside SumatraPDF it makes it harder to access unlock.

However if I leave SumatraPDF  in "presentation" mode go for coffee and screen saving lock is activated.
 I need to same as any other time or application provide login password and then presentation continues.

next time try F5 key to see if it exits from presentation and there is a setting in advanced options to try use ESC in presentation mode
so try set `EscToExit = false` by replace as `EscToExit = true` then next lock up try press esc first.

**Comment by datenschvtz** [comment]
Other Applications like Powerpoint oder PDF Viewer do block the locking when you are in presentation mode. Same is true if you stream something in a browser window. So it should be possible that SumatraPDF could do that too.

**Comment by GitHubRulesOK** [comment]
I am not having any problem 

I let screen saver run while in presentation mode  and on screen saver functioning press esc screen saver to see presentation and can press any mouse or key to continue!

Also as mentioned when screen lock is activated I add my password to resume SumatraPDF presentation without any problems.
